### PR TITLE
[14.0] [ADD] `partner_duns`

### PR DESCRIPTION
**Pull request by ivantodorovich**
This module let's you set a DUNS number on your contacts.

**What's a DUNS Number?**

The **The Data Universal Numbering System**, abbreviated as **DUNS** or **D-U-N-S**, is a proprietary system developed and managed by [Dun & Bradstreet (D&B)](https://www.dnb.com/duns-number.html) that assigns a unique numeric identifier to a single business entity.

It was introduced in 1963 to support D&B's credit reporting practice. It is standard worldwide.

### CODE REVIEW

File: partner_duns/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    duns = fields.Char(
+        help="Data Universal Numbering System",
```
Comment (bodedra) [inline partner_duns/models/res_partner.py]:
```suggestion
    duns = fields.Char(
        string="DUNS",
        help="Data Universal Numbering System",
```

**Review by bodedra — APPROVED** [review]
Functional Review 👍 

Suggest minor correction.

File: partner_duns/models/res_partner.py (hunk 1 above)
Comment (ivantodorovich) [reply partner_duns/models/res_partner.py]: Totally agree, thanks for the suggestion!
I've just changed it

**Review by bodedra — APPROVED** [review]
LGTM

**Review by jcoux — APPROVED** [review]

**Pull request merged**